src/features/mutation_minigame/nodes/dna/Nucleotide.gd

```diff
@@ -0,0 +1,65 @@
+extends PanelContainer
+
+@export_enum("A", "G", "T", "C", "BLANK") var base: String = "BLANK"
+signal mutation(node, mutation)
+@export var bg_visible: bool = true:
```

Comment:
Just a general preference by me: can we put all of our variable exports at the top, then our signals below that, and then our member functions? I can put that change into the code standards if you like that change.

Reply:
Sounds good to me!

Reply:
Update on this, I actually found that Godot has a recommended standard for this as well: https://docs.godotengine.org/en/stable/tutorials/scripting/gdscript/gdscript_styleguide.html#code-order